website/source/netlify-redirects

```diff
@@ -0,0 +1,22 @@
+# Consul Redirects
```

Comment:
So if I understand correctly we'll have to maintain these redirects in both this file as well as in Fastly. Is that right?

Reply:
For now yes, but I might do some investigating and see if we can combine the two into one redirects file and manage it all in one place.

Reply:
Cool, seems pretty low priority but would be nice I guess.